src/App.vue

```diff
@@ -1,14 +1,33 @@
 <template>
 	<div id="app">
-		<QueryBuilder/>
+		<QueryBuilder v-if="isi18nLoaded"/>
 	</div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
 import QueryBuilder from '@/components/QueryBuilder.vue';
+import i18n from 'vue-banana-i18n';
 
 export default Vue.extend( {
+	data() {
+		return {
+			isi18nLoaded: false as boolean,
```

Comment:
isn't just 'false' enough here ?

Reply:
It seems not, TypeScript was complaining.